### DISCUSSION TITLE
Pumpkin Support?

### DISCUSSION THREAD

**Discussion by JiveyGuy** [nasa/cFS#592] [General]
Has anyone used cFS on a Pumpkin C&DH board ([for the Cube Sat Kit](https://www.pumpkinspace.com/store/p24/CubeSat_Kit_for_Microchip_PIC24_MCUs_w%2FUSB_OTG.html))? If so what was your process to get started?

**Comment by BenJWeber** [comment]
OP's coworker here: For reference, the particular chip in our CubeSat Kit is the Microchip dsPIC33FJ256GP710A.
